bin/upload-docs

```diff
@@ -2,22 +2,24 @@
 set -ef -o pipefail
 
 bucket="chain-staging.chain.com"
-if [ "$1" = "prod" ];
+if [ "$2" = "prod" ];
 then
   bucket="chain.com"
 fi
 
 bucket_url=s3://$bucket
 
+git checkout $1-stable
```

Comment:
This has a small issue when the script is called with no arguments. It attempts to run:

```
git checkout -stable
```

Which gives an error when it tries to parse `-s` as a flag to git:

```
error: unknown switch `s'
usage: git checkout [<options>] <branch>
...
```

Let's check that there is at least one argument at the start of the script.